README.md

```diff
@@ -8,38 +8,44 @@
 [![Build Status](https://circleci.com/gh/pyodide/pyodide.png)](https://circleci.com/gh/pyodide/pyodide)
 [![Documentation Status](https://readthedocs.org/projects/pyodide/badge/?version=latest)](https://pyodide.readthedocs.io/?badge=latest)
 
-Python with the scientific stack, compiled to WebAssembly.
+Python and the scientific stack, compiled to WebAssembly.
 
 ## What is Pyodide?
 
-**Pyodide** brings the Python 3.8 runtime to the browser via WebAssembly, along with the Python scientific stack including NumPy, Pandas, Matplotlib, SciPy, and scikit-learn. The [`packages` directory](https://github.com/pyodide/pyodide/tree/master/packages) lists over 75 packages which are currently available. In addition it's possible to install pure Python wheels from PyPi.
+**Pyodide** brings the Python 3.8 runtime to the browser via WebAssembly, along
```

Comment:
I find the bold text here a bit distracting. What do you think about removing it?

Reply:
Fine with me.